modules/emr_node/emr_node.post_update.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @file
+ * Entity Meta Relation node post update file.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Fix possibly misconfigured target_bundles values in some fields.
+ */
+function emr_node_post_update_00001(&$sandbox) {
```

Comment:
Sandbox variable is not needed and the update misses the return type.

Reply:
done